addons/base/views.py

```diff
@@ -316,8 +316,9 @@ def get_auth(auth, **kwargs):
                 # default to most recent version if none is provided in the response
                 version = int(data['version']) if data.get('version') else filenode.versions.count()
                 try:
-                    fileversion = FileVersion.objects.filter(
-                        basefilenode___id=file_id,
+                    basefilenode = BaseFileNode.objects.filter(
+                        _id=file_id).get()
```

Comment:
Could you add to the analysis ticket what the difference in `explain`s for this are? I didn't know this might work differently.